framework/Models/Model.py

```diff
@@ -382,37 +383,38 @@ def submit(self, myInput, samplerType, jobHandler, **kwargs):
     ## works, we are unable to pass a member function as a job because the
     ## pp library loses track of what self is, so instead we call it from the
     ## class and pass self in as the first parameter
-    jobHandler.addJob((self, myInput, samplerType, kwargs), self.__class__.evaluateSample, prefix, metadata=metadata, modulesToImport=self.mods, uniqueHandler=uniqueHandler)
+    jobHandler.addJob((self, myInput, samplerType, kwargs), self.__class__.evaluateSample, prefix, metadata=metadata, modulesToImport=self.mods, uniqueHandler=uniqueHandler, forceUseThreads=forceThreads)
 
-  def submitAsClient(self, myInput, samplerType, jobHandler, **kwargs):
-    """
-        This will submit an individual sample to be evaluated by this model to a
-        specified jobHandler as a client job. Note, some parameters are needed
-        by createNewInput and thus descriptions are copied from there.
-        @ In, myInput, list, the inputs (list) to start from to generate the new
-          one
-        @ In, samplerType, string, is the type of sampler that is calling to
-          generate a new input
-        @ In,  jobHandler, JobHandler instance, the global job handler instance
-        @ In, **kwargs, dict,  is a dictionary that contains the information
-          coming from the sampler, a mandatory key is the sampledVars' that
-          contains a dictionary {'name variable':value}
-        @ Out, None
-    """
-    prefix = kwargs['prefix'] if 'prefix' in kwargs else None
-    uniqueHandler = kwargs['uniqueHandler'] if 'uniqueHandler' in kwargs.keys() else 'any'
+  #def submitAsClient(self, myInput, samplerType, jobHandler, **kwargs):
+    #"""
+        #This will submit an individual sample to be evaluated by this model to a
+        #specified jobHandler as a client job. Note, some parameters are needed
+        #by createNewInput and thus descriptions are copied from there.
+        #@ In, myInput, list, the inputs (list) to start from to generate the new
+          #one
+        #@ In, samplerType, string, is the type of sampler that is calling to
+          #generate a new input
+        #@ In,  jobHandler, JobHandler instance, the global job handler instance
+        #@ In, **kwargs, dict,  is a dictionary that contains the information
+          #coming from the sampler, a mandatory key is the sampledVars' that
+          #contains a dictionary {'name variable':value}
+        #@ Out, None
+    #"""
+    #prefix = kwargs.get("prefix") 
+    #uniqueHandler = kwargs.get("uniqueHandler",'any')
+    #forceThreads = kwargs.get("forceThreads",False)
```

Comment:
Can we add a comment explaining why we are saving this material, and on what conditions it can be removed?

Reply:
removed